composer.json

```diff
@@ -3,7 +3,7 @@
     "description": "provides a role-based access control management",
     "license": "BSD-3-Clause",
     "keywords": [
-        "zf2",
+        "zf3",
```

Comment:
We stop using version with zf components, so I'd suggest to use only `zf` and add please `zendframework` (please see maintainers repository and templates there)